src/test/resources/log4j2.properties

```diff
@@ -0,0 +1,37 @@
+# Set root logger level to error
+#log4j.rootLogger=${app.log.root.level}, Console, File
+rootLogger.level=INFO
+rootLogger.appenderRef.stdout.ref=ConsoleAppender
+rootLogger.appenderRef.rolling.ref=RollingAppender
+
+###### Console appender definition #######
+# All outputs currently set to be a ConsoleAppender.
+# Direct log messages to stdout
+appender.console.type=Console
+appender.console.name=ConsoleAppender
+appender.console.layout.type=PatternLayout
+# use log4j NDC to replace %x with tenant domain / username
+appender.console.layout.pattern=%d{ISO8601} %x %-5p [%c{3}] [%t] %replace{%m}{[\r\n]+}{}%n
+#log4j.appender.Console.layout.ConversionPattern=%d{ABSOLUTE} %-5p [%c] %m%n
+
+###### File appender definition #######
+# Direct log messages to a log file
+appender.rolling.type=RollingFile
+appender.rolling.name=RollingAppender
+appender.rolling.fileName=./target/reports/alfresco-tas.log
+appender.rolling.filePattern=./target/reports/alfresco-tas.log.%d{yyyy-MM-dd}
+appender.rolling.layout.type=PatternLayout
+appender.rolling.layout.pattern=%d{yyyy-MM-dd} %d{ABSOLUTE} %-5p [%c] [%t] %replace{%m}{[\r\n]+}{}%n
+appender.rolling.policies.type = Policies
+appender.rolling.policies.time.type = TimeBasedTriggeringPolicy
+appender.rolling.policies.time.interval = 1
+
+# TestRail particular log file
+# Direct log messages to a log file
+appender.TestRollingAppender.type=RollingFile
+appender.TestRollingAppender.name=TestRollingAppender
+appender.TestRollingAppender.fileName=./target/reports/alfresco-testrail.log
+appender.TestRollingAppender.layout.type=PatternLayout
+appender.TestRollingAppender.layout.pattern=%d{HH:mm:ss} %-5p %c{1}:%L - %m%n
+appender.TestRollingAppender.strategy.type = DefaultRolloverStrategy
+appender.TestRollingAppender.strategy.max = 10
```

Comment:
Pretty much same as the previously highlighted configuration translation issue, I believe we weren't faithful in the translation of the configuration and I'm not sure if this was a desired change or not.